### ISSUE TITLE
Clarification

### ISSUE THREAD

**Issue by aurora-of-earth** [MycroftAI/mimic2#34]
I wonder if this is the repo which is cloned as a submodule by mycroft-core and built for its offline mimic functionality

**Comment by forslund** [comment]
No, it's not this one, it's the [mimic1 repo](https://github.com/MycroftAI/mimic1).

You might also want to checkout the [mimic-packaging repo](https://github.com/MycroftAI/mimic1-packaging)

**Comment by aurora-of-earth** [comment]
The <a href="https://github.com/MycroftAI/mimic1-packaging">mimic1-packaging</a> repo is just dead crap.
- How does an installation of mycroft-core find the mimic1 binaries on the machine?

**Comment by forslund** [comment]
It uses a couple of different approaches,
- Config value with path to the binary
- Check if mimic binary is on path.
